site/src/pages/LoginPage/SignInForm.tsx

```diff
@@ -91,9 +91,9 @@ export const SignInForm: FC<React.PropsWithChildren<SignInFormProps>> = ({
         </div>
       )}
 
-      {Boolean(info) && Boolean(error) && (
+      {Boolean(message) && (
```

Comment:
```suggestion
      {message && (
```

nit: technically this cast is superfluous